src/pages/meeting-rooms.tsx

```
@@ -1,7 +1,148 @@
-import MeetingRooms from "@/components/pages/meeting-rooms";
+import Drawer from "@/components/commons/Drawer";
+import ErrorBoundary from "@/components/commons/ErrorBoundary";
+import useTabDrag from "@/components/commons/Tab/useTabDrag";
+import Layout from "@/components/pages/meeting-rooms/Layout";
+import MeetingRoomsSkeleton from "@/components/pages/meeting-rooms/components/MeetingRoomsSkeleton";
+import ReservationForm from "@/components/pages/meeting-rooms/components/ReservationForm";
+import RoomSelection from "@/components/pages/meeting-rooms/components/RoomSelection";
+import pickedDateAtom from "@/components/pages/meeting-rooms/context/pickedDate";
+import pickedReservationAtom from "@/components/pages/meeting-rooms/context/pickedReservation";
+import { useGetReservations } from "@/components/pages/meeting-rooms/hooks/useGetReservations";
+import Error from "@/components/pages/seats/Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)